package.json

```diff
@@ -65,5 +65,8 @@
         "postcss": "^8.4.38",
         "tailwindcss": "^3.4.4",
         "vite": "^5.3.1"
+    },
+    "peerDependencies": {
+        "@tanstack/react-query": "^5.51.15"
```

Comment:
Should plotly also be in the peer dependency? 

https://github.com/USACE-WaterManagement/groundwork-water/blob/3e01ca9f4d5b70144a5dd54e492c69826f3dcdac/vite.config.js#L16